Mathlib/SetTheory/Cardinal/Basic.lean

```diff
@@ -1355,7 +1355,7 @@ theorem card_le_of_finset {α} (s : Finset α) : (s.card : Cardinal) ≤ #α :=
   @mk_coe_finset _ s ▸ mk_set_le _
 #align cardinal.card_le_of_finset Cardinal.card_le_of_finset
 
--- Porting note: was `simp`. LHS is not normal form.
+-- porting note (#10745): was `simp`. LHS is not normal form.
 -- @[simp, norm_cast]
```

Comment:
This is not quite #10618 but certainly it's a different class of note.

Reply:
Ok @Vierkantor, I'll remove the issue label.

Reply:
Done.